src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,402 @@
+package duke;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+
+public class Duke {
+
+    public static final Path dataDirectory = Path.of(System.getProperty("user.dir") + "\\data");
+
+    public static final Path dataFileDirectory = Path.of(System.getProperty("user.dir") + "\\data" + "\\duke.txt");
+
+    public static final String LINE = "____________________________________________________________";
+
+    public static void writeToDisk(ArrayList<Task> tasks) throws IOException{
+        //Each time I want to write to the Disk, I clear away the entire contents of the data file and write everything
+        //Done by deleting the file and creating it again
+        File clear = new File(String.valueOf(dataFileDirectory));
+        clear.delete();
+        createNewFile();
+
+        //To write to the new file, I will append 1 task from the ArrayList at a time
+        FileWriter fw = new FileWriter(String.valueOf(dataFileDirectory), true);
+        for(Task task : tasks){
+            String temp = "";
```

Comment:
It is probably better to not use "temp" as the name of the intermediate variable. Maybe "newFileContent"?